### ISSUE TITLE
Regression branch loss

### ISSUE THREAD

**Issue by TerryYiDa** [Megvii-BaseDetection/DenseTeacher#7]
你好，代码中求无监督回归loss的时候，loss_deltas = (iou_loss * teacher_quality[b_mask]).mean()，但是teacher_quality有正有负，求出的loss_deltas也会为负，为什么？

**Comment by ZRandomize** [comment]
你好，teacher quality是一个logits（未过sigmoid），所以有正有负，这里代码可能是一个意外，我们的实验结果表明是否将teacher quality乘到iou_loss上不会对结果有什么影响，所以直接使用loss_deptas=iou_loss.mean()即可

**Comment by Joker316701882** [comment]
Please reopen this issue if there is any further question.
